hcloud/routes.go

```diff
@@ -17,6 +20,8 @@ import (
 	"github.com/hetznercloud/hcloud-go/v2/hcloud"
 )
 
+const DefaultClusterCIDR = "10.244.0.0/16"
```

Comment:
Could you move this to `routes_test.go`? This is not a code-default, so I would prefer to have it in the tests to avoid any confusion.